aws/rust-runtime/aws-sigv4/src/http_request/settings.rs

```diff
@@ -39,6 +42,7 @@ pub enum PayloadChecksumKind {
     NoHeader,
 }
 
+/// URI encoding to use while signing
 #[non_exhaustive]
 #[derive(Debug, Eq, PartialEq)]
 pub enum UriEncoding {
```

Comment:
not your code, but maybe we can find a better name for this. `PercentEncodingMode` maybe?

Reply:
I like it! Renamed.